src/utils/GeneralUtils.ts

```diff
@@ -0,0 +1,26 @@
+
+export class GeneralUtils {
```

Comment:
This is something we should avoid. Having a GeneralUtils object is an anti pattern called The God Object.
You can read more about this topic here: https://linearb.io/blog/what-is-a-god-class

Reply:
Don't try to make everything an object. You can keep these kind of stuff functions. It's not wrong to have functional code in your project.